internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+	atlasEntity := srsdk.AtlasEntityWithExtInfo{
+		ReferredEntities: nil,
+		Entity: srsdk.AtlasEntity{
+			Attributes: map[string]interface{}{"description": description, "qualifiedName": qualifiedName},
+			TypeName:   "kafka_topic",
+		},
+	}
+	_, err := srClient.DefaultApi.PartialUpdateByUniqueAttributes(srContext,
+		&srsdk.PartialUpdateByUniqueAttributesOpts{AtlasEntityWithExtInfo: optional.NewInterface(atlasEntity)})
+	return err
+}
+
+func resolveSchemaType(contentType string) string {
+	if strings.Contains(contentType, "avro") {
+		return "AVRO"
+	} else if strings.Contains(contentType, "json") {
+		return "JSON"
+	} else {
+		return "PROTOBUF"
+	}
+}
+
+func registerSchema(details *accountDetails, topicName string, components Components) (int32, error) {
+	// Registering the schema
+	// Subject name follows the TopicNameStrategy
+	subject := topicName + "-value"
+	output.Printf("Registering schema under the subject %s\n", subject)
+	if components.Messages != nil && components.Messages[strcase.ToCamel(topicName)+"Message"].Payload != nil {
+		schema, err := yaml.Marshal(components.Messages[strcase.ToCamel(topicName)+"Message"].Payload)
+		if err != nil {
+			return 0, err
+		}
+		jsonSchema, err := yaml3.ToJSON(schema)
+		if err != nil {
+			return 0, fmt.Errorf("error in converting schema to JSON format: %v", err)
+		}
+		id, _, err := details.srClient.DefaultApi.Register(details.srContext, subject,
+			srsdk.RegisterSchemaRequest{
+				Schema:     string(jsonSchema),
+				SchemaType: resolveSchemaType(components.Messages[strcase.ToCamel(topicName)+"Message"].ContentType),
+			})
+		if err != nil {
+			return 0, fmt.Errorf("unable to register schema due to %v", err)
+		}
+		log.CliLogger.Infof("Schema registered under subject %s with ID %d.\n", subject, id.Id)
+		return id.Id, nil
+	}
+	return 0, fmt.Errorf("schema payload not found in yaml input file")
+}
+
+func updateSubjectCompatibility(details *accountDetails, compatibility string, subject string) error {
+	// Updating the subject level compatibility
+	log.CliLogger.Infof("updating the Subject level compatibility to %s.\n", compatibility)
+	updateReq := srsdk.ConfigUpdateRequest{Compatibility: compatibility}
+	config, _, err := details.srClient.DefaultApi.UpdateSubjectLevelConfig(details.srContext, subject, updateReq)
+	if err != nil {
+		return fmt.Errorf("failed to update subject level compatibility: %v", err)
+	}
+	log.CliLogger.Infof("subject level compatibility updated to %s", config.Compatibility)
+	return nil
+}
+
+func addSchemaTags(details *accountDetails, components Components, topicName string, schemaId int32) ([]srsdk.Tag, []srsdk.TagDef, error) {
+	// Schema level tags
+	var tagConfigs []srsdk.Tag
+	var tagDefConfigs []srsdk.TagDef
+	if components.Messages != nil {
+		for _, tag := range components.Messages[strcase.ToCamel(topicName)+"Message"].Tags {
+			tagDefConfigs = append(tagDefConfigs, srsdk.TagDef{
+				EntityTypes: []string{"cf_entity"},
```

Comment:
What does "cf_entity" mean? Can you explain with a comment or using a constant with a descriptive name?

Reply:
"cf_entity" is a superclass for the tag entity types. Tags of this type can be attached to topics, schemas, etc.
https://docs.confluent.io/cloud/current/stream-governance/stream-catalog-rest-apis.html#searching